people/baruchiro.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "ברוך אודם",
+  "github": "baruchiro",
+  "twitter": "https://twitter.com/baruchiro",
+  "linkedin": "https://www.linkedin.com/in/baruch-rothkoff/",
+  "stackoverflow": "https://stackoverflow.com/users/839513/baruchiro",
+  "website": "https://bscstudent.netlify.app/",
+  "description": "מתכנת, אוהב ומשתדל לתרום לקוד פתוח, אבל לא תרמתי לDOM"
```

Comment:
Hi! Thanks for you contribution ❤️

It will be great if you could write a little bit more about how you are, what are you excited about, hobbies...

Reply:
@MichalPorag
מקווה שזה טוב. אני לא גולש על גלים בזמן החופשי, אני מתכנת וזה התחביב שלי

Reply:
זה מעולה :)